server/test/mockDb/incident.js

```diff
@@ -1,204 +1,204 @@
-        imageurl: 'https://www.image.com',
-        videoUrl: 'https://www.video.com',
-        comment: 'Thanks to NGO for taking care of the less privilege' //comments
-    },
-    {
-        id: 3,
-        createdOn: '2018-11-23',
-        createdBy: 3, // represents the user who created this record  
-        type: 'intervention',       // [red-flag, intervention] 
-        location: '6H 3E',   // Lat Long coordinates   ​ 
-        status: 'draft',     // [draft, under investigation, resolved, rejected] 
-        imageurl: 'https://www.image.com',
-        videoUrl: 'https://www.video.com',
-        comment: 'Let us create awareness for child abuse' //comments
-    },
-    {
-        id: 4,
-        createdOn: '2018-11-23',
-        createdBy: 4, // represents the user who created this record  
-        type: 'intervention',       // [red-flag, intervention] 
-        location: '6H 3E',   // Lat Long coordinates   ​ 
-        status: 'draft',     // [draft, under investigation, resolved, rejected] 
-        imageurl: 'https://www.image.com',
-        videoUrl: 'https://www.video.com',
-        comment: 'Nigerian Government is less concern in Education' //comments
-    }
+  {
+    id: 1,
+    createdOn: '2018-11-23',
+    createdBy: 1,
+    type: 'red-flag', // [red-flag, intervention]
+    location: '6H 3E', // Lat Long coordinates
+    status: 'draft', // [draft, under investigation, resolved, rejected]
+    imageurl: 'https://www.image.com',
+    videoUrl: 'https://www.video.com',
+    comment: 'Human right must be respected'
+  },
+  {
+    id: 1,
+    createdOn: '2018-11-23',
+    createdBy: 1, // represents the user who created this record
+    type: 'red-flag', // [red-flag, intervention]
+    location: '6H 3E', // Lat Long coordinates
+    status: 'draft', // [draft, under investigation, resolved, rejected]
+    imageurl: 'https://www.image.com',
+    videoUrl: 'https://www.video.com',
+    comment: 'Thanks to NGO for taking care of the less privilege'
+  },
+  {
+    id: 3,
+    createdOn: '2018-11-23',
+    createdBy: 3, // represents the user who created this record
+    type: 'intervention', // [red-flag, intervention]
+    location: '6H 3E', // Lat Long coordinates
+    status: 'draft', // [draft, under investigation, resolved, rejected]
+    imageurl: 'https://www.image.com',
+    videoUrl: 'https://www.video.com',
+    comment: 'Let us create awareness for child abuse'
+  },
+  {
+    id: 4,
+    createdOn: '2018-11-23',
+    createdBy: 4, // represents the user who created this record
+    type: 'intervention', // [red-flag, intervention]
+    location: '6H 3E', // Lat Long coordinates
+    status: 'draft', // [draft, under investigation, resolved, rejected]
+    imageurl: 'https://www.image.com',
+    videoUrl: 'https://www.video.com',
+    comment: 'Nigerian Government is less concern in Education'
+  }
 
 ];
 
 
 export const inValidIncidentDate = [
-    /**
+  /**
```

Comment:
These leading comments might not be necessary.

Reply:
Implemented and resolved